### PR TITLE
bring back sort option in typescript-express-reviews app

### PR DESCRIPTION
**Pull request by vkarpov15**
Don't merge until https://github.com/stargate/stargate-mongoose/pull/79 is merged and released. Also relies on #19. With this PR, the typescript-express-review app's tests can now rely on sort order again.

### CODE REVIEW

File: typescript-express-reviews/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "dotenv": "16.0.1",
     "express": "4.x",
     "mongoose": "7.x",
-    "stargate-mongoose": "0.2.0-ALPHA-2"
+    "stargate-mongoose": "file:../../stargate-mongoose/stargate-mongoose-0.2.0-ALPHA-2.tgz"
```
Comment (jeffreyscarpenter) [inline typescript-express-reviews/package.json:22]: did you intend to include this? kind of looks like a local change for testing.

Comment (vkarpov15) [reply typescript-express-reviews/package.json:22]: Yep this is for local testing. Should not merge this branch until there's a compatible release of stargate-mongoose released.

**Comment by vkarpov15** [comment]
@kathirsvn @jeffreyscarpenter I bumped the stargate-mongoose version across all the sample apps, no more custom tarball hacks. Can you please re-review this?

**Review by kathirsvn — APPROVED** [review]
LGTM

**Review by jeffreyscarpenter — APPROVED** [review]

**Pull request merged**